### ISSUE TITLE
service.running not found in salt 2015.8.3

### ISSUE THREAD

**Issue by nmadhok** [saltstack/salt#29768]
I had a salt state that worked fine before I upgraded to the latest open source salt version. 

Here's the state:
```
Install and run service 'at' (job scheduler):
  pkg.installed:
    - name: at
  service.running:
    - name: atd
    - enable: True
    - watch: 
      - pkg: Install and run service 'at' (job scheduler)
```

Here's the error I get:
```
----------
          ID: Install and run service 'at' (job scheduler)
    Function: service.running
        Name: atd
      Result: False
     Comment: State 'service.running' was not found in SLS 'test'
              Reason: 'service' __virtual__ returned False
     Started: 
    Duration: 
     Changes:   
```

Here's versions report:
```
    Salt Version:
               Salt: 2015.8.3
     
    Dependency Versions:
             Jinja2: unknown
           M2Crypto: 0.20.2
               Mako: Not Installed
             PyYAML: 3.11
              PyZMQ: 14.5.0
             Python: 2.6.6 (r266:84292, Jul 23 2015, 05:13:40)
               RAET: Not Installed
            Tornado: 4.2.1
                ZMQ: 4.0.5
               cffi: Not Installed
           cherrypy: Not Installed
           dateutil: Not Installed
              gitdb: Not Installed
          gitpython: Not Installed
              ioflo: Not Installed
            libnacl: Not Installed
       msgpack-pure: Not Installed
     msgpack-python: 0.4.6
       mysql-python: Not Installed
          pycparser: Not Installed
           pycrypto: 2.6.1
             pygit2: Not Installed
       python-gnupg: Not Installed
              smmap: Not Installed
            timelib: Not Installed
     
    System Versions:
               dist: oracle 6.7 
            machine: x86_64
            release: 3.8.13-98.4.1.el6uek.x86_64
             system: Oracle Linux Server 6.7 
```

Notice that even the Jinja2 version says unknown even though Jinja is installed on the server.

**Comment by ssgward** [comment]
@nmadhok - thanks for the find. We'll look into this.

**Comment by ssgward** [comment]
@nmadhok - what was your previous version?

**Comment by nmadhok** [comment]
```
root@test-master:~# salt '*' service.status salt-minion

minion1:
    'service' __virtual__ returned False
minion2:
    'service' __virtual__ returned False
minion3:
    'service' __virtual__ returned False
minion4:
    'service' __virtual__ returned False
minion5:
    'service' __virtual__ returned False
ERROR: Minions returned with non-zero exit code
```

**Comment by nmadhok** [comment]
@ssgward Salt Enterprise 4.0.2

**Comment by ssgward** [comment]
@nmadhok - did you upgrade from 4.0.2 to 2015.8.3 on the same box or is the 2015.8.3 instance a new and separate install?

**Comment by nmadhok** [comment]
@ssgward Tried both. Initially upgraded from 4.0.2 to 2015.8.3 on the same server and to reproduce the issue, built new servers with 2015.8.3 on them.

**Comment by ssgward** [comment]
@nmadhok - were you using Oracle Linux before and after upgrade.

We can't duplicate on Ubuntu or RH or CentOS with 2015.8.3. Wondering if this is an OS specific issue....

**Comment by nmadhok** [comment]
@ssgward Yes I was. I'm getting packages from http://repo.saltstack.com/yum/rhel6/ instead of epel.

**Comment by nmadhok** [comment]
@ssgward https://github.com/saltstack/salt/commit/fdaa81ccf813ef3497a4705ebea882b6b9a5e32e need to be ported to 2015.8.3 to address the issue I highlighted above. Services cannot be restarted or managed until then on Oracle Linux or similar OS.

**Comment by cachedout** [comment]
We have merged #29435 into 2015.8, so this issue should be resolved.
